integration/monai/examples/spleen_ct_segmentation_local/jobs/spleen_ct_segmentation_local/app/config/config_fed_client.json

```diff
@@ -0,0 +1,64 @@
+{
+  "format_version": 2,
+
+  "executors": [
```

Comment:
config part looks good to me.

Since we are moving things inside the "jobs" folder, we might want to move other job folders inside as well.

That could be done in a separate PR.

@holgerroth can you double confirm if we move all "job folder" inside a common "jobs" is good for you?

Reply:
yes, sure.